configure

```diff
@@ -47,6 +47,34 @@ EOF
 }
 
 
+set_gerbil_gcc () {
+	echo "\n;; Added auto via ./configure\n(set! %default-gerbil-gcc \"$1\")" >> \
+	     ./src/gerbil/compiler/driver.ss
```

Comment:
Please don't. It is not acceptible to modify any module of the compiler -- that's bad juju.